src/main/scala/rocket/CSR.scala

```diff
@@ -7,6 +7,7 @@ import Chisel._
 import Chisel.ImplicitConversions._
 import chisel3.withClock
 import freechips.rocketchip.config.Parameters
+import freechips.rocketchip.devices.debug.DebugModuleKey
```

Comment:
This is definitely the dark side of the `implicit p` pattern; we're going to feel the pain if we ever have more than one debug unit and want them to differ according to any of the parameters used here. I am not really sure how to make it diplomatic though, is the only extant direct wire between the core and debug unit that one interrupt line?

Reply:
Yes -- a debug interrupt, a TL slave port from cbus, and a TL master port to fbus.  I guess if there is more than one DM in the address space, then the one whose interrupt is attached is the one this Tile will use and the debug exception address should be customized to that one. This PR doesn't _add_ to the problem -- before, it was a hardcoded 0x800 which didn't allow multiple DMs either.